homeassistant/components/qnap/sensor.py

```diff
@@ -340,6 +326,35 @@ def extra_state_attributes(self):
             return {ATTR_MEMORY_SIZE: f"{size} {UnitOfInformation.GIBIBYTES}"}
 
 
+class QNAPSystemSensor(QNAPSensor):
```

Comment:
Move this back down after `QNAPNetworkSensor`.
No need to move it - it makes reviewing harder.